src/popper/runner.py

```diff
@@ -225,6 +225,7 @@ def _prepare_environment(self, step, env={}):
                     "GIT_COMMIT": self._config.git_commit,
                     "GIT_BRANCH": self._config.git_branch,
                     "GIT_SHA_SHORT": self._config.git_sha_short,
+                    "GIT_REMOTE_ORIGIN_URL": scm.get_remote_url(self._config.repo),
```

Comment:
in order to be consistent with the way in which the other variables (commit, branch, etc) are defined/used, we need to obtain this information as part of the `ConfigLoader.load(https://github.com/getpopper/popper/blob/master/src/popper/config.py)` function, in particular [here](https://github.com/getpopper/popper/blob/master/src/popper/config.py#L56). In this way, we only read the value instead of obtaining the URL, as it is done for the other variables.